metrics/prometheus-agent/values.yaml

```diff
@@ -21,7 +25,7 @@ prometheus:
       name: prometheus-agent-coralogix
       queueConfig:
         capacity: 2500
-        maxSamplesPerSend: 1000
+        maxSamplesPerSend: 500
```

Comment:
if we lower it, how will it affect the remote write ? 
will it require less resources, will it be slower ?

Reply:
It will depend on the amount of data shipped, since this is a repository we're offering to our customers, we should not define how many samples they are going to send or how many shards they need to have, I'm changing by default values we have on Prometheus.

At Coralogix we're going to figure out what's the proper value for our use case.

Reply:
i understand, agree